### DISCUSSION TITLE
Is it possible to disable automatic checks (the code underlined in red or yellow in the buffer) without disabling lsp?

### DISCUSSION THREAD

**Discussion by xoich** [zed-industries/zed#14913] [Feature Requests]
I find these errors distracting, as they are constantly changing as you type. I'm fine with it being something I do manually.

**Comment by elijh** [comment]
I would like to know this too! Except I have the opposite problem: the error underlines have vanished for me and I can't figure out how to get them back. I didn't change anything, but now they are gone. (In markdown using codebook spell checker. The LSP is working, and it still identifies errors but it just wont show the underline).

**Comment by SomeoneToIgnore** [comment]
Yes, since recently landed https://github.com/zed-industries/zed/pull/30316

there's a setting and a temporary toggle in the editor menu:

![image](https://github.com/user-attachments/assets/2d867e82-c2c6-4626-9722-a7102da3adf9)
